python/rpdk/java/templates/generate/HandlerWrapper.java

```diff
@@ -108,8 +115,35 @@ public void testEntrypoint(
             response = ProgressEvent.defaultFailureHandler(e, HandlerErrorCode.InternalFailure);
         } finally {
             writeResponse(outputStream, response);
+            outputStream.close();
+        }
+    }
+    {% else %}
+    public static void main(String[] args) throws IOException {
+        if (args.length != 1){
+            System.exit(1);
+        }
+        final String outputFile = (UUID.randomUUID().toString() + ".txt");
+        try(FileOutputStream output = new FileOutputStream(outputFile)){
+            try(InputStream input=IOUtils.toInputStream(args[0],"UTF-8")){
+                new ExecutableHandlerWrapper().handleRequest(input, output);
+                output.flush();
+            }
+        }
+        readFileToSystemOut(outputFile);
+    }
+
+    private static void readFileToSystemOut(final String fileName) throws IOException {
+        //Create object of FileReader
+        final FileReader inputFile = new FileReader(fileName);
+        try(BufferedReader bufferReader = new BufferedReader(inputFile)) {
+            String line;
+            while ((line = bufferReader.readLine()) != null)   {
+                System.out.println(line);
```

Comment:
What if the handler using the stdout to print something in the middle? Will it impact the result of handler execution?

If so, you mostly need to put the resource with [some boundary lines](https://www.w3.org/Protocols/rfc1341/7_2_Multipart.html) like?
```
==Cloudformation resource result, boundary=boundary{UUID}
....
--boundary{UUID}-- 

```

Reply:
The reason why I write to a file and then read it is specifically so that the response will be easy to regex and there will not be any logs in between the start/end response tags. The last line will look like this:

```
StartResponse-{RESPONSE_HERE}-EndResponse
```

I don't think boundary lines with a UUID will benefit this further

Reply:
Got it. But do you think the boundary logic should be put here? The wrapper is the same logic to write the response to an output stream.

like 
```
System.out.println("__CFN_RESOURCE_START_RESPONSE__");
readFileToSystemOut(fileName);
System.out.println("__CFN_RESOURCE_END_RESPONSE__");
```

Reply:
Yea that makes sense to move it to the handlerwrapper class